.github/workflows/upload_windows_client_to_s3.yml

```
@@ -40,12 +42,13 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        run:
+        run: |
           declare -a FILES=(
-            Outline-Client.exe
-            latest.yml
+            "Outline-Client.exe"
+            "latest.yml"
           )
-          for file in "${FILES[@]}"; do
-            aws s3 cp "client/${file}" s3://outline-releases/client/"${file}"
-            aws s3 cp "client/${file}" s3://outline-releases/client/windows/"${file}"
+          for file in "${FILES[@]}"
+          do
```

Style: https://google.github.io/styleguide/shellguide.html#s5.4-loops
```suggestion
          for file in "${FILES[@]}"; do
```